<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoveCustomerWindowClass</class>
 <widget class="QMainWindow" name="RemoveCustomerWindowClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>201</width>
    <height>69</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remove Customer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="CancelButton">
    <property name="geometry">
     <rect>
      <x>-4</x>
      <y>40</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RemoveButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QLabel" name="IDLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IDTextBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
